### DISCUSSION TITLE
Add functionality for the function keys F13-F24.

### DISCUSSION THREAD

**Discussion by Blogames** [opentrack/opentrack#1911] [Ideas]
It'd be a cool feature for people with a lot of keybinds and peripherals with macro support.

**Comment by GO63-samara** [comment]
https://github.com/opentrack/opentrack/issues/1012
https://github.com/opentrack/opentrack/issues/1269
https://github.com/opentrack/opentrack/discussions/1611
https://github.com/opentrack/opentrack/discussions/1781

**Reply by Blogames** [reply]
Yeah right, cool that they didn't show up for me anywhere. Thanks for explaining.
